src/Sylius/Bundle/AdminBundle/Resources/views/Order/History/addressIndex.html.twig

```diff
@@ -0,0 +1,24 @@
+{% import 'SyliusUiBundle:Macro:table.html.twig' as table %}
```

Comment:
This template is a partial, so should be ``@SyliusAdmin/Order/History/_address.html.twig``.